ceph.spec.in

```diff
@@ -12,6 +12,10 @@
 %bcond_with selinux
 %endif
 
+# LTTng-UST enabled on Fedora, RHEL 6+, and SLES 12
+%if 0%{?fedora} || 0%{?rhel} >= 6 || 0%{?suse_version} == 1315
```

Comment:
>  Thinking of something like
>
> %if 0%{?fedora} || (0%{?rhel} >= 6 && 0%{!?centos}) || 0%{?suse_version} == 1315

@dmick so you suggest we do disable this option in case of centos? but centos can also use the EPEL repo, and hence have access to the lttng and babeltrace packages, right?